vendor/bat-native-ledger/src/bat/ledger/internal/promotion/promotion.h

```diff
@@ -87,6 +101,16 @@ class Promotion {
       const std::string promotion_string,
       ledger::ResultCallback callback);
 
+  void ClaimedTokensSaved(
+      const ledger::Result result,
+      const std::string promotion_string,
+      ledger::ResultCallback callback);
+
+  void ClaimTokensSaved(
+      const ledger::Result result,
+      const std::string& promotion_string,
+      ledger::ResultCallback callback);
+
```

Comment:
Two definitions of the same function only differening by an `&`. Seems like a mistake to me, I would just get rid of the one missing the `&`.

Reply:
one is `Claimed` other is `Claim`, will fix ref missing

Reply:
fixed